builders/build/eth-api/verify-contracts/api-verification.md

```diff
@@ -0,0 +1,530 @@
+            compilerversion: $('#compilerversion').val(),   // see https://etherscan.io/solcversions for list of support versions
+            optimizationUsed: $('#optimizationUsed').val(), //0 = No Optimization, 1 = Optimization used (applicable when codeformat=solidity-single-file)
+            runs: 200,                                      //set to 200 as default unless otherwise  (applicable when codeformat=solidity-single-file)        
+            constructorArguements: $('#constructorArguements').val(),   //if applicable
+            evmversion: $('#evmVersion').val(),             //leave blank for compiler default, homestead, tangerineWhistle, spuriousDragon, byzantium, constantinople, petersburg, istanbul (applicable when codeformat=solidity-single-file)
+            licenseType: $('#licenseType').val(),           //Valid codes 1-14 where 1=No License .. 14=Business Source License 1.1, see https://etherscan.io/contract-license-types
+            libraryname1: $('#libraryname1').val(),         //if applicable, a matching pair with libraryaddress1 required, can include up to 10 external libraries
+            libraryaddress1: $('#libraryaddress1').val()    //if applicable, a matching pair with libraryname1 required, can include up to 10 external libraries
+            libraryname2: $('#libraryname2').val(),         //if applicable, matching pair required
+            libraryaddress2: $('#libraryaddress2').val(),   //if applicable, matching pair required
+            //...
+        },
+        success: function (result) {
+            console.log(result);
+            if (result.status == "1") {
+                //1 = submission success, use the guid returned (result.result) to check the status of your submission.
+                // Average time of processing is 30-60 seconds
+                document.getElementById("postresult").innerHTML = result.status + ";" + result.message + ";" + result.result;
+                // result.result is the GUID receipt for the submission, you can use this guid for checking the verification status
+            } else {
+                //0 = error
+                document.getElementById("postresult").innerHTML = result.status + ";" + result.message + ";" + result.result;
+            }
+            console.log("status : " + result.status);
+            console.log("result : " + result.result);
+        },
+        error: function (result) {
+            console.log("error!");
+            document.getElementById("postresult").innerHTML = "Unexpected Error"
+        }
+    });
+    ```
+
+Upon successful submission, a GUID will be returned as a part of the result. This GUID can be used to check for the submission status. 
+
+=== "Moonbeam"
+    ```bash
+    curl https://api-moonbeam.moonscan.io/api
+            ?module=contract
+            &action=checkverifystatus
+            &guid=GUIDFromResponse
+            &apikey=YourApiKeyToken
```

Comment:
to be consistent across the docs site, i think this should be changed to GUID-FROM-RESPONSE-HERE and YOUR-API-KEY-TOKEN-HERE

Reply:
same with following examples